src/MainWindow.cpp

```diff
@@ -232,7 +233,9 @@ void MainWindow::connectActions()
     // "Treemap" menu
 
     connect( _ui->actionShowTreemap, SIGNAL( toggled( bool )   ),
-	     this,		     SLOT  ( showTreemapView() ) );
+         this,		     SLOT  ( showTreemapView() ) );
```

Comment:
I haven't checked this in the real file, just be aware that GitHub does not display indentation properly when there are tabs or mixture of spaces and tabs...